lib/ansible/modules/cloud/vmware/vmware_cluster_vsan_stretched.py

```diff
@@ -0,0 +1,348 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+#
+# Copyright 2019 VMware, Inc.
+# SPDX-License-Identifier: GPL-3.0-or-later
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import (absolute_import, division, print_function)
+__metaclass__ = type
+
+from ansible.module_utils.vmware import PyVmomi, vmware_argument_spec, find_hostsystem_by_name, find_cluster_by_name
```

Comment:
Could you please create a separate PR for this module ?

Reply:
That was a PEBCAK...
This module should be in a different branch only...